cmd/promtool/unittest_test.go

```diff
@@ -14,11 +14,12 @@
 package main
 
 import (
-	"testing"
-
-	"github.com/stretchr/testify/require"
-
+	"bytes"
+	"fmt"
+	"github.com/grafana/regexp"
 	"github.com/prometheus/prometheus/promql/promqltest"
+	"github.com/stretchr/testify/require"
+	"testing"
```

Comment:
Linter complains about the ordering of imports, see https://github.com/prometheus/prometheus/actions/runs/9920970887/job/27408406132?pr=14461